.travis.yml

```diff
@@ -40,6 +40,12 @@ env:
   - NPM_TAG=latest IMAGE_NAME=theia-https NODE_VERSION=10 ENV_VARS="-e token=" PORT=10443
   - NPM_TAG=next IMAGE_NAME=theia-https NODE_VERSION=10 ENV_VARS="-e token=" PORT=10443
 
+before_install:
+- nvm install 10
+- nvm use 10
+- npm install -g eclint
+- eclint check "**/*"
```

Comment:
When we run `eclint` this way, we will fail CI if any formatting issue is detected. I think it might be better to just warn about it and let the CI continue. Maybe something like this might work (not tested): 
`eclint check "**/*" | echo "Warning: some formatting issues detected. Will let CI continue but please fix ASAP!"

Reply:
I will verify it off my fork to warn instead of failing the build.
I'm only worried that if we see the build passing we won't think to look at the logs to see if any formatting issues have occurred (during a PR review).

Reply:
@marcdumais-work I updated the PR to only warn about potential formatting issues.
I created a sample PR on my fork with formatting errors and the CI still passes:
https://travis-ci.com/vince-fugnitto/theia-apps/builds/138566206#L214